### PR TITLE
Replace subbiali with obtained user name

### PR DESCRIPTION
**Pull request by SF-N**

### CODE REVIEW

File: daint/.bashrc [hunk 1]
```diff
@@ -109,36 +109,36 @@ if ! shopt -oq posix; then
...
-export PATH="/users/subbiali/go1.19.2/bin:/users/subbiali/gitlfs_build/bin:$PATH"
+export PATH="/users/"$USER"/go1.19.2/bin:/users/"$USER"/gitlfs_build/bin:$PATH"
 
 # locale
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
 
 # project
-export PROJECT=/project/s299/subbiali
+export PROJECT=/project/s299/"$USER"
```
Comment (stubbiali) [inline daint/.bashrc]: The upper-level folder `s299` needs to be parameterized too, e.g.:
```suggestion
project_id=$(sacct --format=Account --noheader | head -n 1 | awk '{$1=$1}1')
export PROJECT=/project/"$project_id"/"$USER"
```

File: daint/setup_ghex.sh
Comment (stubbiali) [inline daint/setup_ghex.sh:1]: I think we can get rid of this file, but let's keep it here.

**Review by stubbiali — APPROVED** [review]
Thanks a lot @SF-N. One minor suggestion, but otherwise GTG.

File: daint/.bashrc (hunk 1 above)
Comment (SF-N) [reply daint/.bashrc]: Thanks, I made changes respectively.

**Pull request merged**